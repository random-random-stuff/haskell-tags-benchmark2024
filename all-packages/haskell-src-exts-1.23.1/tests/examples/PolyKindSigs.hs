{-# LANGUAGE GADTs          #-}
--{-# LANGUAGE KindSignatures #-}
{-# LANGUAGE PolyKinds      #-}

module Test where

data Foo :: * -> * where
  Foo :: Foo a
