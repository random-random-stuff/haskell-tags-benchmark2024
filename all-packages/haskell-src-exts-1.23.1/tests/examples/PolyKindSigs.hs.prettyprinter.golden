{-# LANGUAGE GADTs #-}
{-# LANGUAGE PolyKinds #-}
module Test where

data Foo :: * -> * where
        Foo :: Foo a
